### ISSUE TITLE
Update all Wiki pages for the BSD+Patent license change with SPDX identifiers (Bugzilla Bug 1656)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/tianocore.github.io#40]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 1656
Date: 2019-03-22T20:37:55+00:00
From: @mdkinney
To:   @mdkinney
CC:   @jljusten, @lersek, @mdkinney

Depends on:   #1373
Last updated: 2019-04-19T00:18:00+00:00

**Comment by tianocore-issues** [comment]
## Comment 6896
Date: 2019-03-22 20:37:55 +0000
From: @mdkinney

- Industry Specification: ---
- Release Observed: EDK II Master
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): @mdkinney

Update Wiki pages to state that BSD+Patent is the preferred license for the EDK II project.  Update all example commit messages to remove the Contributed-under tag.

**Comment by tianocore-issues** [comment]
## Comment 7006
Date: 2019-04-04 12:08:08 +0000
From: @lersek

*** Bug 1681 has been marked as a duplicate of this bug. ***

**Comment by tianocore-issues** [comment]
## Comment 7121
Date: 2019-04-12 18:55:13 +0000
From: @mdkinney

Assign to owner

**Comment by tianocore-issues** [comment]
## Comment 7122
Date: 2019-04-12 18:55:59 +0000
From: @mdkinney

Patch emails sent for review

https://edk2.groups.io/g/devel/topic/wiki_patch_0_2_change/31070014
https://edk2.groups.io/g/devel/topic/wiki_patch_1_2_remove/31070015
https://edk2.groups.io/g/devel/topic/wiki_patch_2_2_change/31070016

**Comment by tianocore-issues** [comment]
## Comment 7132
Date: 2019-04-15 16:58:18 +0000
From: @mdkinney

Expand the number of places to fix to include the TianoCore web site
and the EDK II C Coding Standards Specification.

**Comment by tianocore-issues** [comment]
## Comment 7133
Date: 2019-04-15 16:59:26 +0000
From: @mdkinney

Website patch email for review

https://edk2.groups.io/g/devel/topic/website_patch_change/31192884

EDK II C Coding Standards Specification patch email for review

https://edk2.groups.io/g/devel/topic/edk2_ccodingstandardsspecification/31193035

**Comment by tianocore-issues** [comment]
## Comment 7208
Date: 2019-04-19 00:11:53 +0000
From: @mdkinney

EDK II C Coding Standards Specification patch V2 email for review

https://edk2.groups.io/g/devel/message/39258

**Comment by tianocore-issues** [comment]
## Comment 7210
Date: 2019-04-19 00:18:00 +0000
From: @mdkinney

Web Site commits:

https://github.com/tianocore/tianocore.github.io/commit/365ebb1b0cc43d9b89df89f24b2344acc40225b4


EDK II C Coding Standards Specification commits:

https://github.com/tianocore-docs/edk2-CCodingStandardsSpecification/commit/d096859f15b9cde19c230ac391ae21ee409af48c

Wiki Commits:

23198b5 Fix typo in link to BSD Plus Patent License page
5439443 Fix typo in link to BSD Plus Patent License page
6a26be5 Change preferred license of EDK II to BSD+Patent
ee27b0b Remove Contributed-under tags
